content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -117,12 +121,74 @@ kind: KubeletConfiguration
 address: "192.168.0.8"
 ```
 
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+address: "192.168.0.8"
+```
+Additionally, you can change the slices/lists values of the kubelet configuration. For example, you can override the `clusterDNS` and `featureGates` fields as follows:
+
+Main kubelet configuration file contents:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.9"
+  - "192.168.0.8"
+featureGates:
+  MemoryQoS: true
+```
+
+Contents of a file in `--config-dir` directory:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+clusterDNS:
+  - "192.168.0.2"
+  - "192.168.0.3"
+  - "192.168.0.5"
+featureGates:
+  KubeletTracing: false
+```
+
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.2"
+  - "192.168.0.3"
+  - "192.168.0.5"
+featureGates:
+  MemoryQoS: true
+  KubeletTracing: false
+```
+
 On startup, the kubelet merges configuration from:
 
-* Command line arguments (lowest precedence).
-* the kubelet configuration
+* Feature gates specified over the command line (lowest precedence).
+* The kubelet configuration.
 * Drop-in configuration files, according to sort order.
-* Feature gates specified over the command line (highest precedence).
+* Command line arguments (highest precedence).
```

Comment:
it's hard to understand how feature gates from the command line has lowest precedence and command line - highest? Maybe say here that this excludes feature gates

Reply:
Sure, I will add that information.